### ISSUE TITLE
Bulk invitation designed

### ISSUE THREAD

**Issue by avivapinchas** [ParabolInc/parabol#6685]
## Issue - Enhancement

Some users will move in a bunch of their users at once - perhaps a whole team or a whole division - when they commit to a tool. Today, they'd invite folks by either sharing the link with everyone (which expires in 30 days) or manually typing in each email address. The latter can be cumbersome if you have a long email address or more than, say, 5 emails to write. 

We should design a way to bulk upload users. 

### Acceptance Criteria (optional)
Users can:
   - Do this
   - Do that
   - Cannot do that

**Estimated effort:** X points ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by jordanh** [comment]
Marking as discussion until clear AC are defined

**Comment by jordanh** [comment]
@avivapinchas I'm going through some of the discussion issues, so dropping some comments here as I do

A long while back (2016 or 2017 perhaps) we ran some observation exercises with users who were trying to invite their team. We noted a common behavior was to go to an existing calendar invitation or email thread in order to copy a list of emails they knew were reflective of the team of folks they wanted to invite. Over the years we made our email address parser smarter to handle more kinds of pasted content (i.e. Outlook Email and Calendar differs from GMail or GCal).

Later we added invite by link, which proved more popular.

Not saying that we can't reduce friction more. However, I did want to have this history here for reference should somebody come take this issue forward to design
